indico_toolkit/auto_populate/populator.py

```diff
@@ -30,28 +31,27 @@ def __init__(self, client: IndicoClient):
         self.client = client
         self.structure = Structure(client)
         self._exceptions = []
-        self._fp = FileProcessing()
 
     def create_auto_classification_workflow(
         self,
-        directory_path: str,
+        directory_path: Union[str, PathLike[str]],
         dataset_name: str,
         workflow_name: str,
         teach_task_name: str,
-        accepted_types: Tuple[str] = (
-            "pdf",
-            "tiff",
+        accepted_types: Tuple[str, ...] = (
             "csv",
             "doc",
             "docx",
-            "png",
             "jpeg",
             "jpg",
+            "pdf",
```

Comment:
should we further expand this list? (eml, msg, tif)

Reply:
Good idea. I've added the net new file types from the [supported file types list](https://docs.indicodata.ai/articles/#!user-guides-6-5/supported-file-types): `eml`, `msg`, `svg`, `tif`.